docs/guides/release-notes.md

```diff
@@ -12,17 +12,46 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
 
 ---
 
-``Last updated: R3.2.1 (UserConnect-7.2.4996)``
+``Last updated: R8.0.0``
 
 ---
+## 8.0.0
+### Versioning Scheme Update
+Starting with this release, the Terraform provider will synchronize its version with the Aviatrix Controller version. This means the provider version has jumped from v3.2.2 to v8.0.0 to align with the Controller’s latest major version. This change makes it easier to determine which provider version is compatible with which Controller version.
+Moving forward, the provider will follow semantic versioning (major.minor.patch).
 
-## 8.0
 ### Notes:
```

Comment:
do we need to include: https://github.com/AviatrixSystems/terraform-provider-aviatrix/commit/1d4cc4d9f756121ded185ae072a673ed1c567131

Reply:
Also bgp multihop? https://github.com/AviatrixSystems/terraform-provider-aviatrix/commit/c44da56b90cb84630c695892e0037d899cb3fec0